Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:2.7
+
+ADD . /opt/ashlar
+WORKDIR /opt/ashlar
+
+RUN apt-get update
+RUN apt-get -y autoremove
+RUN apt-get install -y libgeos-dev binutils libproj-dev gdal-bin
```

Comment:
Consolidating these may end up working better for the AUFS file system because each `RUN` ends up being a separate layer.